packages/headless/src/features/case-assist/case-assist-analytics-selectors.test.ts

```diff
@@ -217,6 +217,25 @@ describe('case assist analytics selectors', () => {
       });
     });
 
+    it('should return the classification matching the specified ID with the field autoSelection when the autoSelection parameter is set to true', () => {
+      const classification = caseAssistCaseClassificationSelector(
+        buildStateWithCaseField(),
+        'service-suggestion-id',
+        true
+      );
+
+      expect(classification).toMatchObject({
+        autoSelection: true,
```

Comment:
Before you can add `autoSelection` here, you'll need to update the [CaseAssistClient](https://github.com/coveo/coveo.analytics.js/blob/master/src/caseAssist/caseAssistClient.ts#L53) in coveo.analytics.js in order to correctly forward the value to the collect event.

Reply:
I'm using the  logSelectFieldSuggestion from coveo.analytics.js not logUpdateCaseField.  you mean updating the  SelectFieldSuggestionMetadata interface in the [logSelectFieldSuggestion](https://github.com/coveo/coveo.analytics.js/blob/88676968178236ddbc1365487d96f1ccfcc7e715/src/caseAssist/caseAssistClient.ts#L61) right? 
I'm currently adding the autoSelection information into this metadata.

Reply:
My bad, I pointed the wrong Case Assist Client method. `logSelectFieldSuggestion` is indeed the method that needs to be modified.

Reply:
> The code look good, but `coveo.analytics.js` has to be modified first.
coveo.analytics.js Updated👌